torch/linalg/__init__.py

```diff
@@ -16,11 +16,15 @@
                          "For a version of this function that does not synchronize, see :func:`{}`.")
 }
 
+def _set_module_and_doc_str(fn, doc_str):
```

Comment:
We should probably use a variant of this function instead of `_add_docstr` at most places. It might be nice to make this a common utility function but errr I didn't want to pass a module arg for every function

Reply:
Sounds good!

Small nit if you want to avoid having to change all the functions in the file, you can set this new function as `_add_docstr` locally :D